src/main/markdown/release-notes.md

```diff
@@ -59,10 +59,79 @@ The GWT Release Notes
 
 <a id="Release_Notes_Current"></a>
 
-<!-- ## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+- Partial support for Java 8 standard library APIs (see below for full list).
+- Fix memory leak with Java 8 compilation.
+- Source level set to Java 8.
+- Static and default methods in interfaces aren't visible to generators. If you want to take advantage of those Java-8isms, you're encouraged to switch to an annotation processor. This could break existing build if an interface is changed to turn a non-default method into a default method.
 
 ### Deprecations
-- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'. -->
+- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'.
+- Deprecate JSR 303 Bean Validation support (javax.validation, c.g.g.validation). The code hasn't been maintained for years.
+
+### Breaking changes
+- GIN needs a workaround now: gwtproject/gwt#9311
+- JSNI disallows method references to Double, String, Boolean and its super types: gwtproject/gwt#9356
+
+### Compiler
+- Update JS reserved keywords to ES6.
+- Fix instanceof for primitives & string.
+- Make exceptions work across Java/JavaScript boundary
+- Allow casting native arrays to Object[].
+- Use patched JDT to circumvent memory leak when compiling for Java 8.
+
+### JsInterop
+- Implement @JsOverlay with default methods in Java8.
+- Split JsInterop annotations into their own JAR when deploying to Maven:
+      <dependency>
```

Comment:
Not sure if this is only with GitHub's rendering, or also with our own processing when publishing the site, but that XML is interpreted as HTML (search for _jsinterop-annotations_ in https://github.com/gwtproject/gwt-site/pull/181/files?short_path=7720a45#diff-7720a459d79ed963bc87695cfe650f0d).